README.md

```diff
@@ -364,9 +364,22 @@ Code changes can readily be verified by running the growing collection of includ
 
 Wasabi can readily be packaged as installable *rpm* or *deb* distributions and deployed at scale as follows:
 
+> Package by running integration tests 1st:
+
+```bash
+% ./bin/wasabi.sh start package
```

Comment:
@bargenilesh Yes, we discovered with @abhishekjain88 the 'package' command by default runs integration tests. So it assumes a running docker container with wasabi, started with 'start' command. 

If we're not running integration tests, then there's no need to start containers explicitly.